app/controllers/equipment_controller.rb

```diff
@@ -33,7 +33,7 @@ def edit
   # POST /lab/1/lab_spaces/1/equipment
   # POST /lab/1/lab_spaces/1/equipment.json
   def create
-    @equipment = @equipment_scope.new(equipment_params)
+    @equipment = @equipment_scope.new(equipment_params.merge(creator_param))
```

Comment:
creo que preferiría usar alguna gema como [audited](https://github.com/collectiveidea/audited) o hacer algo como:
```suggestion
    @equipment = @equipment_scope.new(equipment_params)
    @equipment.creator = current_user
```